README.md

```diff
@@ -253,7 +253,7 @@ from mobly.controllerse import android_device
 class HelloWorldTest(base_test.BaseTestClass):
 
   def setup_class(self):
-    self.ads = self.register_controller(android_device)
+    self.ads = self.register_controller(android_device, min_number=2)
```

Comment:
I'd think it's pretty self explanatory?
People can always read the code or docs (when that exists)...